File: ExerciciosJavaScript(LógicadeProgramacao)/exercicios pag 46(Manzano)/js/exercicioL.js
/*l) Elaborar um programa que efetue a leitura de valores positivos inteiros até que um valor negativo
seja informado. Ao final devem ser apresentados o maior e o menor valores informados pelo
usuário. */

var numero = 0
var contadora = 0
var guardaNumero = 0

while (contadora < 1) {
    numero = parseInt(prompt("Digite um numero"))

    if (numero > guardaNumero) {
        guardaNumero = numero
    }

    if (numero < 0) {
        contadora = contadora + 1
        alert("O menor numero é o: "+numero)
    }
}

alert("O maior numero é o "+guardaNumero)

File: Java/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/manzanoPag46/ExercicioL.java
package manzanoPag46;

import java.util.Scanner;

/*l) Elaborar um programa que efetue a leitura de valores positivos inteiros até que um valor negativo
seja informado. Ao final devem ser apresentados o maior e o menor valores informados pelo
usuário. */

public class ExercicioL {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		int numero = 0;
		int contadora = 0;
		int guardaNumero = 0;

		while (contadora < 1) {
			System.out.println("Digite um numero");
			numero = sc.nextInt();

			if (numero > guardaNumero) {
				guardaNumero = numero;
			}

			if (numero < 0) {
				contadora = contadora + 1;
				System.out.println("O menor numero é o: " + numero);
			}
		}

		System.out.println("O maior numero é o " + guardaNumero);

		sc.close();

	}

}
